src/transformers/models/vision_encoder_decoder/modeling_flax_vision_encoder_decoder.py

```diff
@@ -393,7 +393,7 @@ def encode(
         Example:
 
         ```python
-        >>> from transformers import FlaxVisionEncoderDecoderModel
+        >>> from transformers import FlaxVisionEncoderDecoderModel, ViTFeatureExtractor
```

Comment:
```suggestion
        >>> from transformers import ViTFeatureExtractor, FlaxVisionEncoderDecoderModel
```
Sorry, this seems more logical to me :)

Reply:
Make sense to me :-)
(surprisingly but gladly, `make style` is happy with this order. Outside the doc example, it will change the order back!)